samples/csharp_dotnetcore/30.asp-mvc-bot/Asp-Mvc-Bot/wwwroot/lib/jquery/LICENSE.txt

```diff
@@ -0,0 +1,36 @@
+Copyright jQuery Foundation and other contributors, https://jquery.org/
```

Comment:
I'm not clear on why this is here. Isn't JQuery downloaded off the CDN? We don't want it checked intout our codebase.